pyproject.toml

```diff
@@ -9,7 +9,7 @@ license = "MIT"
 python = "^3.8,<3.11"
 
 [tool.poetry.group.dev.dependencies]
-demisto-sdk = "1.10.6"
+demisto-sdk = {git = "https://github.com/demisto/demisto-sdk.git", rev="efbad5491db83783bd410dfe20f0b815945bef2b"}
```

Comment:
please revert

Reply:
Of course. I'll revert pyproject.toml and poetry.lock

Reply:
Of course. Both files reverted.